Makefile.am

```diff
@@ -122,6 +122,12 @@ EXTRA_DIST = \
 	OVS_WITH_P4_ARCH.rst \
 	CONTRIBUTING_P4OVS.rst \
 	OVS-WITH-P4-BUILD-README \
+	p4proto/kctrl/README.md \
```

Comment:
Why are we adding these files here?
shouldn't these files be added as part of EXTRA_DIST in p4proto/automake.mk?

Reply:
Since these are README's I have added them in global Makefile, let me know if any concerns with this